### ISSUE TITLE
Renaming a ride with the rides window open empties the rides window

### ISSUE THREAD

**Issue by RugnirViking** [OpenRCT2/OpenRCT2#6602]
**OS:** Windows 10
**Version:** 0.1.2
**Commit/Build:** 3125956

When renaming a ride with the rides window open the rides window lost all rides and said '0 rides' in the corner.
Tested with vanilla and the issue was not present

- [ ] Reproducible in RCT2 (vanilla)?
- [x] Multiplayer?

**Steps to reproduce:**
1. Open rides window
2. With it still open, rename a ride

**Comment by duncanspumpkin** [comment]
It should just takes a tick to refresh after the name change. Is that not happening for you?

**Comment by rwjuk** [comment]
I concur with @duncanspumpkin - I tested this on several heavily-loaded maps, and the rides window refreshes after a tick (less than half a second usually). Can you upload the save game where you're seeing this?

**Comment by SidaRadis** [comment]
@RugnirViking might be having the same issue I reported in #5804.

**Comment by RugnirViking** [comment]
@rwjuk I tested it a couple times. On a new map it does indeed refresh relatively fast. However as @SidaRadis mentioned it does not refresh while paused. However, when I go back to my heavily loaded map it still continues to stay unrefreshed for a good long while. I got it to slow down specifically by spamming peeps with the large train command and running on hyper speed but I can attack the save file here (note that the savefile scenario has incorrect exits so the peeps get stuck which is how the sitation came to be where I discovered this):

[Autumn Challenge!loaded heavily.zip](https://github.com/OpenRCT2/OpenRCT2/files/1452988/Autumn.Challenge.loaded.heavily.zip)
